tendrl/monitoring_integration/grafana/utils.py

```diff
@@ -16,9 +16,8 @@ def get_conf():
     try:
 
         # Graphite and Grafana will be running on localhost
-        NS.config.data["grafana_host"] = "localhost"
+        NS.config.data["grafana_host"] = "127.0.0.1"
```

Comment:
@r0h4n localhost can be mapped to some other ip, so to be specific I have mentioned 127.0.0.1 instead of localhost
If this is not needed then I would revert it back